README.md

```diff
@@ -6,13 +6,16 @@
 
 Build it:
 ```
-$ docker build -t trackerbot . 
+docker build -t trackerbot .
 ```
+
 Run it interactive:
 ```
-$ docker run -itv $(pwd):/app --name tb trackerbot
+docker run -itv $(pwd):/app --name tb trackerbot
 ```
+
 Run tests inside the container:
 ```
-$ npm run test
+npm install
```

Comment:
In my opinion the Dockerfile is ok (`npm install --production` makes sense)
The only thing to add in the README is to install also the development dependencies with `npm install;` before running the tests. That's all :)